src/services/env/Env.ts

```diff
@@ -102,7 +115,8 @@ export function getPathConfigDefault(apiUrlDefault: string = '') {
     apiUrl: apiUrlDefault,
     version: '2.4.0',
     product: 'Kuma',
-    mode: 'global',
+    mode: 'zone',
+    zone: 'default',
```

Comment:
For the `zone` field in the index.html template, the ticket for reference: https://github.com/kumahq/kuma/issues/8767

> Actually just thought, are the changes in so that we get mode: "zone" here?

Are you asking if `mode: 'zone'` will be in the index.html config object? If so, then yes (see https://github.com/kumahq/kuma/blob/master/pkg/api-server/testdata/index.html).

Or are you asking about the dev setup wiring in kuma-gui that makes it so that setting `KUMA_MODE` in tests/cookies affects the dev server? That also works.

> Ok so from what I've understood, this (this being `zone`) isn't `yet` in our index.html file and we don't know for sure if/when it will be added, so I don't think we should add this here either. This JSON should be an exact replica of the shape that we get in the index.html file.

That makes sense to me. I’ve commented out this addition and the addition of `KUMA_ZONE` with a TODO item mentioning the relevant ticket.

Reply:
> Are you asking if mode: 'zone' will be in the index.html config object? If so, then yes (see https://github.com/kumahq/kuma/blob/master/pkg/api-server/testdata/index.html).

Yeah I was, thanks for the confirmation, the rest that you said all sounds good to me, resolving this one.